app/models/reports/region_summary_schema.rb

```diff
@@ -721,6 +769,22 @@ def diabetes_appts_scheduled_rates(entry)
       })
     end
 
+    memoize def diabetes_treatment_outcome_rates_fasting_and_hba1c(entry, with_ltfu)
+      rounded_percentages({
+        bs_below_200_rates_fasting_and_hba1c: adjusted_diabetes_under_care(:bs_below_200, :fasting)[entry.region.slug][entry.period] + adjusted_diabetes_under_care(:bs_below_200, :hba1c)[entry.region.slug][entry.period],
+        bs_200_to_300_rates_fasting_and_hba1c: adjusted_diabetes_under_care(:bs_200_to_300, :fasting)[entry.region.slug][entry.period] + adjusted_diabetes_under_care(:bs_200_to_300, :hba1c)[entry.region.slug][entry.period],
+        bs_over_300_rates_fasting_and_hba1c: adjusted_diabetes_under_care(:bs_over_300, :fasting)[entry.region.slug][entry.period] + adjusted_diabetes_under_care(:bs_over_300, :hba1c)[entry.region.slug][entry.period],
+        missed_visits_rates: diabetes_missed_visits(with_ltfu: with_ltfu)[entry.region.slug][entry.period],
+        visited_without_bs_taken_rates: visited_without_bs_taken[entry.region.slug][entry.period],
+        bs_below_200_rates_random: adjusted_diabetes_under_care(:bs_below_200, :random)[entry.region.slug][entry.period],
+        bs_below_200_rates_post_prandial: adjusted_diabetes_under_care(:bs_below_200, :post_prandial)[entry.region.slug][entry.period],
+        bs_200_to_300_rates_random: adjusted_diabetes_under_care(:bs_200_to_300, :random)[entry.region.slug][entry.period],
+        bs_200_to_300_rates_post_prandial: adjusted_diabetes_under_care(:bs_200_to_300, :post_prandial)[entry.region.slug][entry.period],
+        bs_over_300_rates_random: adjusted_diabetes_under_care(:bs_over_300, :random)[entry.region.slug][entry.period],
+        bs_over_300_rates_post_prandial: adjusted_diabetes_under_care(:bs_over_300, :post_prandial)[entry.region.slug][entry.period]
+      })
+    end
+
```

Comment:
This method is needed because of how rounded_percentages is implemented. In order for it to calculate percentages correctly, it needs to have a balanced numerator and denominator part. Since in this we are only including fasting and hba1c patient counts in the numerator, but the denominator contains all the measurements, adding the rest of the measurements in the hash makes up for it. 
Marking this as resolved.